### ISSUE TITLE
PageField dropdown empty after other error on same admin page

### ISSUE THREAD

**Issue by Arany** [divio/django-cms#5724]
Steps to reproduce:
* Have a Model with a PageField and another field that can throw an error:
    
```python
from django.db import models
from cms.models.fields import PageField
class ModelA(models.Model):
    sometext = models.CharField(max_length=10)
    page = PageField()
    
from django.contrib import admin
@admin.register(ModelA)
class ModelAAdmin(admin.ModelAdmin):
    pass
```

* Have some Test CMS Pages
* Open the Adminpanel and open "New ModelA"
* Leave the sometext field empty and just click save

-> An error page occurs (because the sometext field is empty), and the PageField Dropdown now only holds an empty "---" Entry:
![bildschirmfoto 2016-10-11 um 11 20 03](https://cloud.githubusercontent.com/assets/793921/19265278/d32e8a5c-8fa4-11e6-86b3-a050699fb4ec.png)

After a reload, it is there again, but obviously all entered user input is gone:
![bildschirmfoto 2016-10-11 um 11 20 15](https://cloud.githubusercontent.com/assets/793921/19265274/cc9ee164-8fa4-11e6-82b1-8fb3f1e9fdcd.png)

This is the easiest way for me to reproduce it, sometimes is also happens already on the very first, not even on the error page.

On Chrome, I get this error in the console: bundle.forms.pageselectwidget.min.js?3.3.2:1 Uncaught TypeError: Cannot read property '_pageSelectWidgets' of undefined

**Comment by czpython** [comment]
Hello @Arany,
Which djangoCMS and Django versions are you using?

**Comment by Arany** [comment]
3.3.2, should I try it with the newest version?

**Comment by czpython** [comment]
Yes please, I believe this was fixed on 3.3.3

**Comment by Arany** [comment]
I can't reproduce it any longer in 3.3.3, thanks for your answer!
